frontend/src/lib/components/LemonModal/LemonModal.tsx

```diff
@@ -55,7 +55,7 @@ export function LemonModal({
     const modalContent = (
         <>
             <div className="LemonModal__closebutton">
-                <LemonButton icon={<IconClose />} size="small" status="stealth" onClick={onClose} />
+                <LemonButton icon={<IconClose />} size="small" status="stealth" onClick={onClose} aria-label="close" />
```

Comment:
Ah! 🙌 

![celebrate](https://user-images.githubusercontent.com/984817/184095857-72f4480b-0825-4514-b53c-bff5299bde4e.gif)

Reply:
You're the legend for noticing (and building the tooling to know that) this is missing!